src/client/components/OrganizationPanel.js

```diff
@@ -1,24 +1,64 @@
-import React from 'react'
-import styled from 'styled-components'
-import { Text } from '@aragon/ui'
+import React, { useContext } from 'react'
 import CardWrapper from '../wrappers/styleWrappers/CardWrapper'
-
-const addMore = () => console.log('add more orgs')
+import { useProfile } from '../hooks'
+import { ModalContext } from '../wrappers/modal'
+import OrganizationTile from './OrganizationTile'
+import { open, removeItem } from '../stateManagers/modal'
+import { Text, theme } from '@aragon/ui'
+import styled from 'styled-components'
 
 const OrganizationPanel = () => {
+  const { organizations, viewMode } = useProfile()
+  const { dispatchModal } = useContext(ModalContext)
+
+  const organizationsNotEmpty = Object.keys(organizations).length > 0
+
+  const cardProps = {
+    title: 'Organizations',
+    addMore: organizationsNotEmpty
+      ? () => dispatchModal(open('organization'))
+      : null,
```

Comment:
Where is `addMore` being used? I don't see it elsewhere in this PR. 

It seems strange to me that one branch of the ternary returns a function, while the other returns `null`. Will this require us to check for `null`, wherever it's used? 

I suspect that we can avoid such checks by moving the ternary inside the function:

```js
addMore: () => organizationsNotEmpty && dispatchModal(open('organization'))
```

Reply:
It actually needs to be function or null, it's used in CardWrapper to determine whether to show "add more" link. Which, thinking about it, could be done better - but I'd leave it for refactoring phase.